api/v1/azureappconfigurationprovider_types.go

```diff
@@ -69,6 +71,12 @@ type AzureAppConfigurationKeyValueOptions struct {
 	Refresh         *DynamicConfigurationRefreshParameters `json:"refresh,omitempty"`
 }
 
+// AzureAppConfigurationFeatureFlagOptions defines the options of fetching feature flags from AppConfiguration.
+type AzureAppConfigurationFeatureFlagOptions struct {
+	Selectors []KeyValueSelector `json:"selectors,omitempty"`
+	Refresh   *RefreshSettings   `json:"refresh,omitempty"`
+}
+
 // KeyValueSelector defines the filters when fetching the data from Azure AppConfiguration
 type KeyValueSelector struct {
```

Comment:
How about rename it to `KeyLabelSelector` ?  Since it's not only for selecting key-values, but also for select feature flags